projects/components/src/data-exporter/data-exporter.component.ts

```diff
@@ -347,11 +360,18 @@ export class DataExporterComponent implements OnInit, OnDestroy {
     }
 
     private getDisplayNameForField(fieldName: string): string {
-        for (const column of this.columns) {
-            if (column.fieldName === fieldName) {
-                return column.displayName || column.fieldName;
-            }
+        if (this.fieldNameMap.has(fieldName)) {
```

Comment:
If your if clause has a return, there's no need for an else. This practice minimizes indentation.

When would this be called with a `fieldName` that doesn't exist? I don't see a need for the else, I think we're not getting coverage for this, I don't think it's possible this is ever hit unless we write a bug.